client/prisonbreak.lua

```diff
@@ -163,10 +163,10 @@ CreateThread(function()
                         inRange = true
                         if securityLockdown then
                             sleep = 0
-                            DrawText3D(Gates[k].coords.x, Gates[k].coords.y, Gates[k].coords.z, "~r~SYSTEM LOCKDOWN")
+                            QBCore.Functions.DrawText3D(Gates[k].coords.x, Gates[k].coords.y, Gates[k].coords.z, "~r~SYSTEM LOCKDOWN")
```

Comment:
Never use this function, add a local function for this instead, using the core function takes more ms

Reply:
@TheiLLeniumStudios please fix this :) we're supposed to be removing that function.

Reply:
okay I wasn't aware. Will remove